src/GitHub.Exports/Helpers/AssemblyResolver.cs

```diff
@@ -51,7 +52,7 @@ static Assembly LoadAssemblyFromRunDir(object sender, ResolveEventArgs e)
             {
                 var requestedName = e.Name.TrimSuffix(".dll", StringComparison.OrdinalIgnoreCase);
                 var name = new AssemblyName(requestedName).Name;
-                if (!ourAssemblies.Contains(name))
+                if (!ourAssemblies.Contains(name, new LambdaComparer<string>((lhs, rhs) => String.Compare(lhs, rhs, StringComparison.OrdinalIgnoreCase))))
```

Comment:
I think you could just pass in `StringComparer.OrdinalIgnoreCase` here rather than the `LambdaComparer`. :smile: